components/Review/CreateEditReview.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  Pressable,
+  SafeAreaView,
+  TextInput,
+} from "react-native";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import { Entypo, Ionicons } from "@expo/vector-icons";
+import { useDispatch } from "react-redux";
+import { useAppSelector } from "../../redux/hooks";
+import { selectUser } from "../../redux/selectors/user";
+import { RatingInput } from "./RatingView";
+import ReviewAPI from "./ReviewAPI";
+import { addReview, editReview } from "../../redux/slices/review";
+import BizReviewHeader from "./BizReviewHeader";
+import { selectBusinessById } from "../../redux/selectors/business";
+import { RootStackParamList } from "../../route-settings";
+import { selectReviewById } from "../../redux/selectors/review";
+import { ReviewType } from "../../src/APITypes";
+
+const styles = StyleSheet.create({
+  back: {
+    flexDirection: "row",
+    alignItems: "center",
+    marginLeft: -3,
+  },
+  backText: {
+    fontFamily: "Mada-SemiBold",
+    fontSize: 18,
+    textAlign: "left",
+    padding: 10,
+    paddingLeft: "1%",
+  },
+  post: {
+    flexDirection: "row",
+    alignItems: "center",
+  },
+  postText: {
+    fontFamily: "Mada-Black",
+    color: "#DA5125",
+    fontSize: 18,
+    textAlign: "right",
+    padding: 10,
+    paddingLeft: "1%",
+  },
+  descriptor: {
+    color: "grey",
+    fontFamily: "Mada-Medium",
+    paddingTop: 10,
+    paddingBottom: 8,
+  },
+});
+
+interface BackButtonProps {
+  action: () => void;
+}
+
+function BackButton({ action }: BackButtonProps) {
+  return (
+    <Pressable onPress={action}>
+      <View style={styles.back}>
+        <Entypo name="chevron-left" size={24} />
+        <Text style={styles.backText}>Back</Text>
+      </View>
+    </Pressable>
+  );
+}
+
+function PostButton({ action }: BackButtonProps) {
```

Comment:
Could probably combine this with backbutton to make like "FunctionalButton" to save some lines and make reusable if you give it a style prop and conditionally render the entypo icon. Just an idea tho, totally not necessary